gwdetchar/io/tests/test_html.py

```diff
@@ -256,6 +291,14 @@ def test_write_flag_html_with_plots(tmpdir):
     shutil.rmtree(str(tmpdir), ignore_errors=True)
 
 
+def test_scaffold_omega_scans():
+    times = [1126259462]
+    channel = 'X1:STRAIN'
+    page = html.scaffold_omega_scans(times, channel)
+    print(page)
```

Comment:
Can this be removed?

Reply:
thanks @duncanmmacleod, I've removed it, this was left over from when I was writing the unit test and hadn't woken up enough yet